INSTALL

```diff
@@ -1,19 +0,0 @@
-You need the following utilities to run rpmlint:
```

Comment:
This information of non-python dependencies is useful to preserve. can you maybe add it to README.md if you want to get rid of INSTALL for some reason? otherwise please keep the file